VideoLocker/res/layout/registration_spinner_item.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<TextView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    style="?android:attr/spinnerItemStyle"
+    tools:targetApi="17"
+    android:singleLine="true"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:ellipsize="marquee"
+    android:textAlignment="inherit"
+    android:fontFamily="sans-serif"
```

Comment:
We want RoboRegular so that the spinner matches with the TextInputLayout (defaults to RobotoRegular). We want that to match the TextInputLayout in the Login activity.